ui/lib/core/addon/components/shamir/flow.js

```diff
@@ -44,7 +43,7 @@ import { tracked } from '@glimmer/tracking';
  */
 export default class ShamirFlowComponent extends Component {
   @service store;
-  @tracked errors = A();
+  @tracked errors = [];
```

Comment:
Should this also be `null` since it's set to that on line 75? It looks like it's only used in the template so the empty array will also be fine as a falsey value. For typing it'd be nice to be consistent, so alternately we could also "reset" the errors in the task below by setting `this.errors = []`